spec/index.bs

```diff
@@ -1776,6 +1783,7 @@ enum GPUFeatureName {
     "depth-clip-control",
     "depth24unorm-stencil8",
     "depth32float-stencil8",
+    "multi-draw-indirect",
```

Comment:
Apologies, it's been a while since we were discussing this and I don't remember exactly what we decided to do here. How does this proposed feature correspond to Vulkan's multiDrawIndirect and drawIndirectCount? Does it require both?

The validation below seems to make multiDrawIndirect() always available, and only gate the drawCountBuffer on "multi-draw-indirect", which would imply it maps to just Vulkan's drawIndirectCount (and probably should be called "draw-indirect-count") and implementations will have to emulate multiDrawIndirect to expose it on all devices. (From skimming #1949 I don't think this was the intent, although IIUC emulating it _is_ fairly easy? hardest part is DrawIndex/`gl_DrawID`)

Reply:
As originally proposed and accepted (held on a rebase over `first-instance-indirect`) multiDrawIndirect() would only be available with both multiDrawIndirect and drawCountBuffer since 30% of Android devices did not support multiDrawIndirect and 60% of Android devices did not support drawCountBuffer. Though these numbers are from gpuinfo.org which does not remove old drivers. Not sure where you get that multiDrawIndirect is always available.

It is true that multiDrawIndirect() could be emulated everywhere and only drawCountBuffer would need to be an optional feature. Though it would not be very useful without `first-instance-indirect` (or some sort of draw ID) so making it core might be somewhat dubious. Though if adding a draw ID then implementing/emulating `multi-draw-indirect` in core is reasonable. If this is the desired route then I can make a new PR to add multi draw capability to core.

The only issue with multi-draw-indirect in core is a silent performance difference when emulating vs native multi draw indirect. If it's a feature the developer will know and be able to emulate the behavior themselves, though developer side emulation is only possible with `first-instance-indirect`, again possibly missing on about 30% of Android devices.

If you think the emulation solution should be explored that has two consequences. First I will start another PR which specs out multi-draw-indirect as core to discuss the issues related to doing that. And second drawCountBuffer becomes a fairly small feature which only has a feature parity difference with multiDrawIndirect of about 30% of Android devices and may not even be worth it at that point.